react/features/base/util/httpUtils.js

```diff
@@ -0,0 +1,27 @@
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * Wrapper around fetch GET requests to handle json-ifying the response
+ * and logging errors.
+ *
+ * @param {string} url - The URL to perform a GET against.
+ * @returns {Promise}
+ */
+export function doGet(url) {
```

Comment:
should it be `doGetJson` ?

Reply:
This seems like a fair request. The response streams can be converted to a few formats, although we only really use json, so specifying json seems explicit.

Reply:
I've named it doGetJSON to be consistent with how prefixURL and prefixID and prefixURI get capitalized. I suppose I could also capitalize it GET...